utility/BUILD.bazel

```diff
@@ -3,6 +3,7 @@ load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
 load("@rules_jvm_external//:defs.bzl", "artifact")
 load("@dagger//:workspace_defs.bzl", "dagger_rules")
 
+'''A kt_android_library() rule that builds utility module kotlin source files into a library.'''
```

Comment:
This looks fine. We can go for `#` as it is just a single line comment. 
It seems like this commit is not visible in your last commit `522993ff7 Added Firebase comment`

Reply:
Done!